### PR TITLE
[5.0] Added "make:seed" Command

### PR DESCRIPTION
**Pull request by Jaspur**

### CODE REVIEW

File: src/Illuminate/Foundation/Console/stubs/seed.stub [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+class {{class}}TableSeeder extends Seeder {
+
+    public function run()
+    {
+        {{class}}::truncate();
+
+        {{class}}::create([]);
+    }
+
+}
```
Comment (GrahamCampbell) [inline src/Illuminate/Foundation/Console/stubs/seed.stub]: needs to end in a new line

File: src/Illuminate/Foundation/Console/stubs/seed.stub [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<?php namespace {{namespace}};
+
+class {{class}}TableSeeder extends Seeder {
+
+    public function run()
```
Comment (GrahamCampbell) [inline src/Illuminate/Foundation/Console/stubs/seed.stub]: Needs to be indented with tabs.

Comment (GrahamCampbell) [inline src/Illuminate/Foundation/Console/stubs/seed.stub]: Also, the run method need s a docblock.

**Comment by Jaspur** [comment]
Didn't see https://github.com/laravel/framework/pull/7324 ; that's a better approach :p

File: src/Illuminate/Foundation/Console/stubs/seed.stub [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+	/**
+	 * Create a seed for {{class}}.
+	 *
+	 * @return void
+	 */
+	public function run()
+	{
+		{{class}}::truncate();
+
+		foreach(range(1, 10) as $index)
```
Comment (GrahamCampbell) [inline src/Illuminate/Foundation/Console/stubs/seed.stub]: Missing space before the opening bracket.

File: src/Illuminate/Foundation/Console/stubs/seed.stub [hunk 4]
```diff
@@ -0,0 +1,22 @@
+<?php namespace {{namespace}};
+
+class {{class}}TableSeeder extends Seeder {
+
+	/**
+	 * Create a seed for {{class}}.
```
Comment (GrahamCampbell) [inline src/Illuminate/Foundation/Console/stubs/seed.stub]: This description isn't very good.

Comment (Jaspur) [inline src/Illuminate/Foundation/Console/stubs/seed.stub]: I know :') But I don't know something else to describe it with.

**Comment by brunowego** [comment]
+1

**Pull request closed without merging**